### DISCUSSION TITLE
Rollback to android-10.3.2 ?

### DISCUSSION THREAD

**Discussion by danielchh** [maplibre/maplibre-native#2952] [Q&A] [answered]
So why were the last two releases a continuation of the 10.3 version? Should I change 11.5.1 to 10.3.3 in my project as well?

**Comment by louwers** [comment, marked as answer]
I would recommend against that!

The 10.x.x release series is intended for those who are not able to upgrade to 11.x.x (yet). We faciliate backporting fixes, but we do not have LTS releases as per our release policy. Some manual testing is done, but we do not have proper CI set up for 10.x.x. 
 https://github.com/maplibre/maplibre-native/wiki/Release-Policy-Android-and-iOS#backporting
